### ISSUE TITLE
[Feature request] Order of class interfaces

### ISSUE THREAD

**Issue by buffcode** [FriendsOfPHP/PHP-CS-Fixer#4108]
Having the following input:

```php
<?php

class Example implements CheckInterface, AuthInterface {}
```

a `ordered_implements` rule should order the import alphanumerically to achieve the following output:

```php
<?php

class Example implements AuthInterface, CheckInterface {}
```

Other orders could be `length` and `none` (identical to `ordered_imports`).

This rule should not be risky, as interfaces cannot overwrite each other when being implemented.

**Comment by dmvdbrugge** [comment]
In the case of `ordered_imports` sorting on `none` has at least the grouping.
Would a `none` option here not have the same effect as just not enabling the fixer?

**Comment by buffcode** [comment]
> Would a `none` option here not have the same effect as just not enabling the fixer?

Maybe to disable the fixer in an subsequent rule?
E.g. the Symfony ruleset enables the fixer but you don't want it, as such you set it explicitly to `none`.

**Comment by dmvdbrugge** [comment]
> > In the case of ordered_imports sorting on none has at least the grouping.
> 
> which is a bug that will be fixed very soon by a PR of mine ;) @ see [#2696 (comment)](https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/2696#issuecomment-296082422)

I was talking about sort function, which in case of your comment is alpha, not none.

**Comment by dmvdbrugge** [comment]
I created a PR for this: #4113 😄
